src/bin/vip-sync.js

```diff
@@ -51,7 +51,7 @@ command( {
 			let bail = false;
 
 			for ( const err of error.graphQLErrors ) {
-				if ( err.message !== 'Site is already syncing' ) {
+				if ( err.message !== 'Site is already syncing.' ) {
```

Comment:
Did we change the error in GOOP? I don't see it in the `trunk`.

```
 vip-go-api % git grep 'Site is already syncing'
lib/modules/data-sync/data-sync.js:                     throw new ValidationError( 'Site is already syncing' );
lib/modules/data-sync/preflight.js:             throw new ValidationError( 'Site is already syncing', 'already_syncing' );
lib/modules/site-actions/import/start.js:                       throw new ValidationError( 'Site is already syncing' );
```

Reply:
I only edited the error message for punctuation, but did not investigate the underlying mechanisms that might throw the error. To be clear, this syncing action would be from a Platform environment to a local environment in case that makes a difference.

Reply:
```suggestion
				if ( err.message !== 'Site is already syncing' ) {
```
We are checking against the error message we received from the upstream. So, to change the error message, we need to change it where it was created. I'd like to propose that we leave this change for another PR where we can find where ends meet and fix it all over the place 🙏

Reply:
Agreed! Work to change the error message should be handled in a separate PR. But I'm glad that my little PR helped you notice that the error message needed to be fixed!